src/main/java/com/training/assignments/miniproject/EmployeeOperation.java

```diff
@@ -0,0 +1,44 @@
+package com.training.assignments.miniproject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeOperation {
+    static List<com.training.assignments.miniproject.Employee> employeeList = new ArrayList<>();
+
+    public void createEmployee(String name, String address, int salary, int id) {
+        employeeList.add(new com.training.assignments.miniproject.Employee(name, address, salary, id));
+    }
+
+    public List<com.training.assignments.miniproject.Employee> getEmployee() {
+        return employeeList;
+    }
+
+    public void deleteEmployee(int id) {
+        Employee employeeToRemove = null;
+        for (Employee emp : employeeList) {
+            if (emp.id == id) {
+                employeeToRemove = emp;
+                break;
+            }
+        }
+        employeeList.remove(employeeToRemove);
```

Comment:
What if we specify an id which is not there in the list?
Won't it give a null pointer error here? 
Please try this scenario and fix if there is an null pointer thrown.